File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    h = ''
    j = ''
    
    
    for i in num_list:
        if i % 2 == 0: # 짝수일 때
            j += str(i)
        else:
            h += str(i)
            
    return int(j)+int(h)

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        
        String hol = "";
        String jjak = "";
        
        for(int i=0; i<num_list.length; i++){
            if(num_list[i] % 2 == 1){
                hol += num_list[i];
            }
            else jjak += num_list[i];
        }
        
        
        int hol_int = Integer.parseInt(hol);
        int jjak_int = Integer.parseInt(jjak);
        
        return hol_int + jjak_int;
        
    }
}